test/index.js

```diff
@@ -0,0 +1,41 @@
+import { it } from 'micro-should';
+
+import './aliascheck.js';
+//import './aliascheckbabyjub.js';
+import './babyjub.js';
+import './babyjub_js.js';
+import './binsub.js';
+import './binsum.js';
+import './comparators.js';
+import './eddsa.js';
+import './eddsa_js.js';
+import './eddsamimc.js';
+import './eddsamimcsponge.js';
+import './eddsaposeidon.js';
+import './escalarmul.js';
+import './escalarmulany.js'; // (Maximum call stack size exceeded)
```

Comment:
what does this mean?

Reply:
```suggestion
import './escalarmulany.js';
```
Fixed.  😬   (was just a leftover note that shouldn't have been committed)